### PR TITLE
Set PHP version to 7.4 in workflow and composer dependency updates

### PR DESCRIPTION
**Pull request by joschahenningsen**
I think this might fix the test/build issues in #27

### CODE REVIEW

**Comment by joschahenningsen** [comment]
Yeah this reduces the warnings but apparently there are still issues (with the php version?) @kordianbruck I don't feel comfortable changing it without your opinion on this :D

**Comment by joschahenningsen** [comment]
Found the cause, GitHub actions switched to php8 as default. 
Setting up the version like I did should work as long as we depend on 7.4.

Should I revert the dependency updates or should we keep this while we're at it?

File: .github/workflows/branches.yml [hunk 1]
```diff
@@ -10,6 +10,10 @@ jobs:
     - uses: actions/checkout@v2
     - name: Validate composer.json and composer.lock
       run: composer validate
+    - name: Setup PHP 7.4
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: '7.4'
```
Comment (kordianbruck) [inline .github/workflows/branches.yml:16]: Shouldn't this be before the composer validate step?

**Comment by joschahenningsen** [comment]
Afaik composer validate checks compatibility between packages and does not really depend on a php version. Probably makes sense to do the setup step first, changing this tomorrow.

**Comment by kordianbruck** [comment]
Sure, but composer also checks the env for extension for example.

**Pull request merged**